### PR TITLE
[TASK] Get rid of "resname" attribute in XLF files

### PR DESCRIPTION
**Pull request by linawolf**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1167
See: https://forge.typo3.org/issues/105647
Releases: main, 13.4, 12.4

### CODE REVIEW

**Comment by liayn** [comment]
The "approved=yes" can be left out too. That's not relevant for Core at all.

File: Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Faq.rst [hunk 1]
```diff
@@ -65,19 +65,7 @@ How to convert to the new language XLIFF file format
 
 If you have :ref:`downloaded an XLIFF file <migrate-from-pootle>` from the
```
Comment (liayn) [inline Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Faq.rst:66]: Pootle Server has been shut down completely. I guess this is obsolete

Comment (linawolf) [reply Documentation/ApiOverview/Localization/TranslationServer/Crowdin/Faq.rst:66]: That should be a separate PR then, as this change is about the Core change

**Comment by linawolf** [comment]
The core change was only about removing resname. Also I believe when a translation is not approved it will not be displayed..

**Comment by liayn** [comment]
Fine with me

**Review by xperseguers — COMMENTED** [review]
looks ok.

**Pull request merged**